README.md

```diff
@@ -93,21 +93,46 @@ e.g. make sure all the different projects in the solution are building for Debug
 
 ## Service Configuration
 
-This app requires some API secrets from external services. Configure them here, if you want to use these services:
+This app requires some API secrets from external services. If you will be using any of these services, follow the instructions in this section to configure them.
 
-### Wink Configuration
-Create winkConfig.ts file under "app\pages\onboarding\winkOnboardingPage" with the following contents.
+> This section is optional. If you are not using any of these external services, you can safely ignore it.
 
-```ts
-    export class WinkConfig {
+### 1. Create Config File
+Create a `config.ts` file under the "app" folder with the following contents. This file is listed in `.gitignore` to help avoid accidental check-in of secrets.
 
-        public static API_ENDPOINT: string = "https://api.wink.com";
-        public static CLIENT_ID: string = "";
-        public static CLIENT_SECRET: string = "";
+```ts
+    export class Config {
+
+        public static Wink: any = {
+            "API_ENDPOINT" : "https://api.wink.com",
+            "CLIENT_ID" : "",
+            "CLIENT_SECRET" : ""
+        };
+
+        public static Auth0: any = {
+            "DOMAIN" : "",
+            "CLIENT_ID" : "",
+            "CLIENT_SECRET" : ""
+        };

```

Comment:
This is good general feedback for this app, not specific to this PR. Can you open an issue? In general, since this is currently just a dev focused app we are asking the dev to provide all the secrets needed to run this app correctly. If we ever release this as a non-dev product then we need to figure out how to handle secrets... your latter suggestion is better, we will need to stash the secrets in a web service since that's the only way to ensure the secrets will not leak.